lib/richmond/rtfm.rb

```diff
@@ -95,8 +97,8 @@ def recording?
     end
 
     def record! line
-      @result.output[@output_filename].push line 
-      @result.input[@input_filename].push line 
+      @result.output[@output_filename].push line
+      @result.input[@the_name_of_the_file_being_read].push line
```

Comment:
I didn't keep pushing in this direction, but when I looked at the code the first time I had trouble understanding what some of these variables were doing.  I renamed this one to remove any ambiguity, and I think if I kept going I'd rename ```output_filename``` too.